src/server/routes.js

```
@@ -47,9 +48,28 @@ const methods = {
     try {
       const hash = await chain.addTx(txBytes)
       return utils.bufferToHex(hash)
-    } catch (e) {
-      console.log(e)
-    }
+    } catch (e) {}
+
+    return null
+  },
+
+  async getTxByHash([txHash]) {
+    let tx = {}
+    try {
+      const obj = await getTxByHash(txHash)
+      if (obj) {
+        tx = obj.toJSON(true)
+      }
+    } catch (e) {}
```

These catch-all exception handlers can mask potential errors

---

Yeah, I'll remove them.